### ISSUE TITLE
Accessibility Features For Those With Visual Disabilities

### ISSUE THREAD

**Issue by spookyswords** [YoYoGames/GameMaker-Bugs#9482]
### Is your feature request related to a problem?

Gamemaker Studio 2 is highly inaccessible to those with low vision. 

### Describe the solution you'd like

A good solution to this is adding layout editing settings — text scaling and icon scaling ootions being the most important of which. Other settings, which are no less essential to making the game engine accessible though I can acknowlwdge may be harder to implement, would be the ability to move locations of different sections and options for better tool organisation to avoid the eye-strain caused by visual clutter, as well as the much-needed support of zooming in and out within the app when a touchscreen monitor is being used. 

### Describe alternatives you've considered

Unfortunately, there are no alternatives. When it comes to accessibility, something either is or isn't — and Gamemaker simply isn't. 

### Additional context

I am an aspiring indie developer with visual impairments, but I found myself disappointed in the lack of accessibility that rendered me entirely unable to even begin creating a game.

**Comment by Fallingstars8** [comment]
^^^^^^^ !!!!!!!!
